ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx

```diff
@@ -0,0 +1,102 @@
+import { ListItem, View } from "native-base";
+import React from "react";
+import { Image, StyleSheet } from "react-native";
+import { OperationTypeEnum as IbanOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/IbanOperationDTO";
+import { OperationTypeEnum as RefundOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/RefundOperationDTO";
+import { OperationTypeEnum as OnboardingOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/OnboardingOperationDTO";
+import { H4 } from "../../../../../components/core/typography/H4";
+import { LabelSmall } from "../../../../../components/core/typography/LabelSmall";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { formatDateAsShortFormat } from "../../../../../utils/dates";
+import { formatNumberAmount } from "../../../../../utils/stringBuilder";
+import { OperationListDTO } from "../../../../../../definitions/idpay/timeline/OperationListDTO";
+import { Icon } from "../../../../../components/core/icons";
+import I18n from "../../../../../i18n";
+
+const styles = StyleSheet.create({
+  alignCenter: {
+    alignItems: "center"
+  },
+  spaceBetween: {
+    justifyContent: "space-between"
+  },
+  sidePadding: {
+    paddingHorizontal: 8
+  },
+  imageSize: { height: 16, width: 24 }
+});
+
+const getHourAndMinuteFromDate = (date: Date) =>
+  `${String(date.getHours()).padStart(2, "0")}:${String(
+    date.getMinutes()
+  ).padStart(2, "0")}`;
+
+type TimelineOperationCardProps = {
+  operation: OperationListDTO;
+};
+
+export const TimelineOperationListItem = ({
```

Comment:
We are using too many "in" checks. Can we just create a different component for each Operation type?